support-workers/src/test/scala/com/gu/zuora/subscriptionBuilders/DigitalSubscriptionBuilderSpec.scala

```diff
@@ -78,7 +79,7 @@ class DigitalSubscriptionBuilderSpec extends AsyncFlatSpec with Matchers {
       readerType shouldBe Gift
       redemptionCode.isDefined shouldBe true
       redemptionCode.get.substring(0, 4) shouldBe "gd03"
-      initialTerm shouldBe 13
+      initialTerm shouldBe DigitalSubscriptionGiftRedemption.expirationTimeInMonths + 1
```

Comment:
I never know how much we should "share" between the tests and code. as the tests are supposed to be independent and the next level of simplicity!  But in this case I'm sure it won't do any harm either way.